backend/Gemfile

```diff
@@ -7,7 +7,7 @@ ruby '2.6.6'
 
 gem 'rails', '~> 6.1.1'
 gem 'pg', '>= 0.18', '< 2.0'
-gem 'puma', '~> 3.12'
+gem 'puma', '~> 4.3'
```

Comment:
Are these updates necessary for this PR? I think we shouldn't need to update it here? Let me know otherwise!

Reply:
The master already has these updated, so when getting the changes from master, it automatically changed the versions.

Reply:
Oh, hmmm :thinking: I think it should not be shown in your PR if it is on master? Did you fetch the upstream and rebase from your fork?